habitat-hitl/habitat_hitl/environment/controllers/llm_controller.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import threading
+from typing import Any, Dict, Union
+
+import numpy as np
+from habitat_llm.agent import Agent
+from habitat_llm.agent.env import EnvironmentInterface
+from habitat_llm.planner.llm_planner import LLMPlanner
+from habitat_llm.utils import fix_config, setup_config
+from habitat_llm.utils.analysis import CodeTimer
```

Comment:
I suppose that this code will never be called if `habitat-llm` is not used, correct?

Reply:
correct! This controller assumes you are using habitat-llm-agent underneath. I can add comment header to explicitly call this out.